rust/src/push/mod.rs

```diff
@@ -568,7 +568,10 @@ impl FilteredPushRules {
             .filter(|rule| {
                 // Ignore disabled experimental push rules
 
-                if !self.msc1767_enabled && rule.rule_id.contains("org.matrix.msc1767") {
+                if !self.msc1767_enabled
+                    && (rule.rule_id.contains("org.matrix.msc1767")
+                        || rule.rule_id.contains("org.matrix.msc3933"))
+                {
```

Comment:
It wouldn't be naturally obvious to me that `msc1767_enabled` controls enables _both_ of these. (I can see that the second explicitly [builds](https://github.com/matrix-org/matrix-spec-proposals/pull/3933/files#diff-891c76ad806a71b4169d7fcdf334d806c83512edb37de49690781c7c9f5bc7d4R19-R21) on the former though.)

I guess we can rip out these experimental guards some day, and this isn't going to live around forever?

Reply:
> It wouldn't be naturally obvious to me that `msc1767_enabled` controls enables _both_ of these. (I can see that the second explicitly [builds](https://github.com/matrix-org/matrix-spec-proposals/pull/3933/files#diff-891c76ad806a71b4169d7fcdf334d806c83512edb37de49690781c7c9f5bc7d4R19-R21) on the former though.)
> 
> I guess we can rip out these experimental guards some day, and this isn't going to live around forever?

Me neither, the comment for the experimental config flag says "MSC1767 and friends"... There's like 4 or 5 other MSCs all related and it is confusing IMO.

So yes, hopefully we burn this with fire at some point?